client/src/partials/records/auxillary_cash_records/auxillary_cash_recors.js

```diff
@@ -165,10 +165,16 @@ angular.module('bhima.controllers')
     }
 
     function convert (){
+      console.log('PPPPPPPPPPPPPPPPPPPPPP');
```

Comment:
Why are you committing these `console.log`'s the development branch?  Are they necessary?